src/app/app.component.html

```diff
@@ -1,5 +1,9 @@
+
 <div id="page-wrap">
     <div id="page-content">
+        <app-header class="head"></app-header>
```

Comment:
header and footer should be outside the page content ? and the page content should be a main tag

Reply:
The main is done on an other pr arriving after we merge this and the categories.